### PR TITLE
Mention feature is added

### PR DESCRIPTION
**Pull request by VTAkbay**

### CODE REVIEW

File: src/bot/create.ts [hunk 1]
```diff
@@ -46,6 +48,13 @@ export async function createPoll(
         channel: channelId,
         text: 'Poll',
         blocks: [
+            {
+                type: "section",
+                text: {
+                    type: "mrkdwn",
+                    text: mentionOwner + " has started a new poll!",
```
Comment (m-ali-akbay) [inline src/bot/create.ts]: This would be more readable:
```ts
text: `<@${userId}> has started a new poll!`
```

**Review by m-ali-akbay — CHANGES_REQUESTED** [review]
It would be good if you could simplfy the code

**Review by m-ali-akbay — APPROVED** [review]
👍 LGTM

**Comment by VTAkbay** [comment]
Bots not adding to database on vote_right.

**Pull request merged**